templates/vue3/src/App.vue

```diff
@@ -0,0 +1,160 @@
+                <p>
+                    By default, when your app runs in "development" mode, it connects
+                    to a test network ("testnet") wallet. This works just like the main
+                    network ("mainnet") wallet, but the NEAR Tokens on testnet aren't
+                    convertible to other currencies - they're just for testing!
+                </p>
+                <p>
+                    Go ahead and click the button below to try it out:
+                </p>
+                <p style="text-align:center; margin-top:2.5em">
+                    <button v-on:click="login">Sign in</button>
+                </p>
+            </main>
+        </template>
+    </div>
+</template>
+
+<script>
+
+import getConfig from './config'
+const nearConfig = getConfig(process.env.NODE_ENV || 'development')
+
+console.log(`networkId:${nearConfig.networkId} CONTRACT_NAME:${nearConfig.contractName}`);
+
+import { login, logout, onSubmit } from './utils'  
+import './global.css'
+
+import Notification from './components/Notification.vue'
+
+    export default {
+        created() {
+            if (this.signedIn) {
+                this.retrieveSavedGreeting()
+            }
+        },
+        name: 'App',
+        components: {
+            Notification
+        },
+        data: function () {
+            return {
+                savedGreeting: "Hello stranger",
+                newGreeting: "Hello stranger",
+                notificationVisible: false,
+                networkId: nearConfig.networkId,
+            }
+        },
+        computed: {
+            signedIn() { return window.walletConnection.isSignedIn() },
+            accountId() { return window.accountId }
+        },
+
+        methods: {
+
+            login() {
+                console.log("calling utils.login")
+                login()
+                this.retrieveSavedGreeting()
+            },
+
+            logout: logout,
+
+            retrieveSavedGreeting()  {
+                //retrieve greeting
+                var self=this
+                window.contract.getGreeting({ accountId: window.accountId })
+                    .then(greetingFromContract => {
+                        self.savedGreeting = greetingFromContract
+                        self.newGreeting = greetingFromContract
+                    })
+            },
+    
+        saveGreeting: async function (event) {
+            this.updating = true;
+            // fire frontend-agnostic submit behavior, including data persistence
+            // look in utils.js to see how this updates data on-chain!
+            await onSubmit(event) //gets data directly from HTML elements
+
+            // update upper `greeting` message with persisted value
+            this.savedGreeting = this.newGreeting;
+            this.updating = false;
+
+            this.notificationVisible = true; //show notification
+            //Notification auto-hides after 10s because tag <aside>.- see: global.css
```

Comment:
without a timed removal from the dom or setting `display:none`, do multiple updates in a row open the notification again? in the other templates there’s a `setTimeout` here

Reply:
There's a single <aside> for notification, multiple updates in a row do not show more than one notification, but you're right, the `setTimeout `is required in order to restart the keyframe animation . **I'll restore the setTimeout.**

Reply:
Yes, this is sort of a fake notification, as we wanted to avoid overcomplicating it, but maybe this was a bad way to simplify it. Do you think there's an easy way to insert a new `<aside>` on every form submission? If you would rather do that, I think this might be an ok place for the Vue template to break with the patterns of the other two.

Reply:
Let's close this issue first as defined. 

After that maybe we can open a new issue to enhance notifications in all templates and I can help with that. 

There's no "easy" way, the `<aside>` is using position:fixed, gotta make that adjustable. In the vanilla template we have to clone & insert into the DOM a new `<aside>` for each notification adjusting position & animation. For Vue we can use https://github.com/euvl/vue-notification and I guess there are similar solutions for React.

Reply:
Yeah, not sure I’d want the notification to be “more real” – it might be fine as-is? Regardless, I agree that it shouldn’t be fixed as part of this PR.